lib/common.js

```diff
@@ -984,6 +984,19 @@ function endsWith(str, suffix) {
   return str.indexOf(suffix, str.length - suffix.length) !== -1;
 }
 
+function findCredentialsByID(id, cb) {
+	exports.doGetApiCall(fhreq.getFeedHenryUrl(), "box/api/credentials", i18n._("Error reading credentials: "), function(err, data) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace all tab characters in this file by sequences of white-spaces. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATabCharacter)